### PR TITLE
fixing postgres related main issues

### PR DESCRIPTION
**Pull request by dhanuka84**

### CODE REVIEW

**Comment by harsha89** [comment]
Hi Dhanuka,

Please change the scripts in the rss manager core accordingly :). 

Thanks,
Harsha

File: components/rss-manager/org.wso2.carbon.rssmanager.core/src/main/java/org/wso2/carbon/rssmanager/core/dto/restricted/RSSInstanceDSWrapper.java [hunk 1]
```diff
@@ -40,7 +42,8 @@
 	private RSSInstance rssInstance;
 
 	private final ConcurrentHashMap<String, RSSDatabaseDSWrapper> dbDSMap = new ConcurrentHashMap<String, RSSDatabaseDSWrapper>();
-
+	private static final String POSTGRES = "postgres";
```
Comment (harsha89) [inline components/rss-manager/org.wso2.carbon.rssmanager.core/src/main/java/org/wso2/carbon/rssmanager/core/dto/restricted/RSSInstanceDSWrapper.java]: It's better to move this in to RSSManagerConstants class :).

File: components/rss-manager/org.wso2.carbon.rssmanager.core/src/main/java/org/wso2/carbon/rssmanager/core/dao/impl/DatabaseUserDAOImpl.java [hunk 2]
```diff
@@ -64,18 +64,19 @@ public void addDatabaseUser(PreparedStatement nativeAddUserStatement, DatabaseUs
...
 			createUserStatement.setString(3, user.getType());
 			createUserStatement.setInt(4, user.getTenantId());
-			if(RSSManagerConstants.RSSManagerTypes.RM_TYPE_SYSTEM.equalsIgnoreCase(user.getType())){
+			/*if(RSSManagerConstants.RSSManagerTypes.RM_TYPE_SYSTEM.equalsIgnoreCase(user.getType())){
 				createUserStatement.setString(5, user.getType());
 			} else {
 				createUserStatement.setString(5, user.getInstances().iterator().next().getName());
-			}
+			}*/
+			createUserStatement.setInt(5, user.getInstances().iterator().next().getId());
```
Comment (harsha89) [inline components/rss-manager/org.wso2.carbon.rssmanager.core/src/main/java/org/wso2/carbon/rssmanager/core/dao/impl/DatabaseUserDAOImpl.java]: Shall we check whether type is system and if it's system we needs to set -1. Now it's taking the first instance ID as user may have multiple instances if type is system

Comment (dhanuka84) [inline components/rss-manager/org.wso2.carbon.rssmanager.core/src/main/java/org/wso2/carbon/rssmanager/core/dao/impl/DatabaseUserDAOImpl.java]: Yep , I missed that. Thanks for pointing this

**Pull request merged**